docs/admin.rst

```diff
@@ -174,3 +174,24 @@ possession when they attempt to log in to SecureDrop.
 .. |Verify Google Authenticator works| image:: images/admin_enter_verification.png
 .. |Enable YubiKey| image:: images/admin_enable_yubikey.png
 .. |Verify YubiKey| image:: images/admin_verify_yubikey.png
+
+Managing `site-specific` updates on teams with multiple admins
```

Comment:
This should be under a new section, called e.g. "Updating the SecureDrop Server" or similar (currently it's under "Adding Users"): 

![screen shot 2017-08-21 at 9 29 31 am](https://user-images.githubusercontent.com/7832803/29529124-42dee310-8653-11e7-967a-1af6a0db3b97.png)